app/models/partner.rb

```diff
@@ -1,6 +1,7 @@
 class Partner < ActiveRecord::Base
   include Initializer
 
+  @@per_page = 5
```

Comment:
Given that (if I remember correctly) osra has about 12 partners?  Then paginating on 6,7 or 13,14 might not be a bad idea.  Probably 14?